### PR TITLE
Add `this-week-in-neovim`

### PR DESCRIPTION
**Pull request by phaazon**
Relates to #505.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -366,7 +366,7 @@ Tree-sitter is a new system introduced in Neovim 0.5 that incrementally parses y
 - [lmburns/kimbox](https://github.com/lmburns/kimbox) - A colorscheme with a dark background, and vibrant foreground that is centered around the color brown. A modification of [Kimbie Dark](https://marketplace.visualstudio.com/items?itemName=dnamsons.kimbie-dark-plus).
 - [rockyzhang24/arctic.nvim](https://github.com/rockyzhang24/arctic.nvim) - A Neovim colorscheme ported from VSCode Dark+ theme with the strict and precise color picking for both the editor and UI.
 - [meliora-theme/neovim](https://github.com/meliora-theme/neovim) - Warm and cozy theme for Neovim.
-- [Yazeed1s/minimal.nvim](https://github.com/yazeed1s/minimal.nvim) - Two tree-sitter supported colorschemes that are inspired by base16-tomorrow-night and monokai-pro. 
+- [Yazeed1s/minimal.nvim](https://github.com/yazeed1s/minimal.nvim) - Two tree-sitter supported colorschemes that are inspired by base16-tomorrow-night and monokai-pro.
```
Comment (phaazon) [inline README.md:369]: Sorry for that diff, probably my `nvim` trailing some whitespaces automatically. I can disable that if it’s a problem.

Comment (tversteeg) [reply README.md:369]: No problem at all.

**Comment by tversteeg** [comment]
Thanks for the PR!

**Comment by phaazon** [comment]
Should be better now.

**Pull request merged**